src/database/data/accommodation-categories.data.ts

```diff
@@ -0,0 +1,29 @@
+import { AccommodationCategory } from 'src/modules/accommodations/entities';
+
+export const accommodationCategories: Pick<AccommodationCategory, 'name'>[] = [
+  { name: 'Hotels' },
```

Comment:
how are you rendering icons? here with lib like react-icons or mui

Reply:
On the frontend we will have something similar to this : 

```js
    const amenityIcons: Record<string, JSX.Element> = {
        "Swimming Pool": <FaSwimmer />,
        "Free WiFi": <FaWifi />,
        "Parking": <FaParking />,
        "Bedroom": <FaBed />,
    };
```

This would be possible because we have predefined categories in the database whose names are unique.